spring-cloud-stream/src/main/java/org/springframework/cloud/stream/binder/CommonBinderProperties.java

```diff
@@ -22,7 +22,7 @@
  *
  * @author Gary Russell
  */
-public interface BinderProperties {
+public interface CommonBinderProperties {
```

Comment:
I think it would be better to rename this something that clarifies it is only a holder for property names (it's not actually properties in the sense of key/value pairs). Perhaps `BinderPropertyKeys`?